ToDo-Garden/ToDoGardenUI/Sources/ToDoGardenUIResource/UIColor+ToDoGarden.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIColor+ToDoGarden.swift
+//
+//
+//  Created by SONG on 3/1/24.
+//
+
+import UIKit.UIColor
+
+extension UIColor {
+  public static let toDoGardenRed = UIColor(resource: .toDoGardenRed)
+  
+  public static let toDoGardenOrange = UIColor(resource: .toDoGardenOrange)
+  
+  public static let toDoGardenYellow = UIColor(resource: .toDoGardenYellow)
+  
+  public static let toDoGardenLeaf = UIColor(resource: .toDoGardenLeaf)
+  
+  public static let toDoGardenOlive = UIColor(resource: .toDoGardenOlive)
+  
+  public static let toDoGardenMint = UIColor(resource: .toDoGardenMint)
+  
+  public static let toDoGardenBlue = UIColor(resource: .toDoGardenBlue)
+  
+  public static let toDoGardenPink = UIColor(resource: .toDoGardenPink)
+  
+  public static let toDoGardenPurple = UIColor(resource: .toDoGardenPurple)
+  
+  public static let toDoGardenBrown = UIColor(resource: .toDoGardenBrown)
+  
+  public static let toDoGardenBlack = UIColor(resource: .toDoGardenBlack)
+  
+  public static let toDoGardenGray = UIColor(resource: .toDoGardenGray)
+  
+  public static let toDoGardenGreenDark = UIColor(resource: .toDoGardenGreenDark)
+  
+  public static let toDoGardenGreenGray = UIColor(resource: .toDoGardenGreenGray)
+  
+  public static let toDoGardenGreenBackground = UIColor(resource: .toDoGardenGreenBackground)
+  
+  public static let toDoGardenGray1 = UIColor(resource: .toDoGardenGray1)
+  
+  public static let toDoGardenGray2 = UIColor(resource: .toDoGardenGray2)
+  
+  public static let toDoGardenGray3 = UIColor(resource: .toDoGardenGray3)
+  
+  public static let toDoGardenEditButtonRed = UIColor(resource: .toDoGardenEditButtonRed)
+  
+  public static let toDoGardenEditButtonBlue = UIColor(resource: .toDoGardenEditButtonBlue)
+  
+  public static let toDoGardenEditButtonYellow = UIColor(resource: .toDoGardenEditButtonYellow)
+  
+  public static let toDoGardenLightPink = UIColor(resource: .toDoGardenLightPink)
```

Comment:
```suggestion
  public static let toDoGardenLightRed = UIColor(resource: .toDoGardenLightRed)
```
피그마 디자인 가이드에 맞추어 토큰 명이 수정되어야할 것 같습니다 :)

Reply:
@noah0316 
반영하였습니다. 
[3325973](https://github.com/ToDo-Garden/ToDo-Garden/pull/34/commits/3325973e1a0aa3e7c5b1d3ea15f7c698877bcd03)

Reply:
@HG-SONG 
확인했습니다 :)